src/main/java/com/dev/cinema/Main.java

```diff
@@ -79,6 +84,19 @@ public static void main(String[] args) {
         String userTwoPassword = "1234";
         authenticationService.register(userTwoEmail, userTwoPassword);
 
-        System.out.println(userService.findByEmail("pawa@gmail.com"));
+        User user = userService.findByEmail("pawa@gmail.com").get();
+        System.out.println(user);
+
+        ShoppingCart shoppingCart = shoppingCartService
+                .getByUser(userService.findByEmail("pawa@gmail.com").get());
+        System.out.println("FIRST " + shoppingCart);
+        shoppingCartService.addSession(movieSession,
+                userService.findByEmail("pawa@gmail.com").get());
+        shoppingCart = shoppingCartService
+                .getByUser(userService.findByEmail("pawa@gmail.com").get());
+        System.out.println("SECOND " + shoppingCart);
```

Comment:
let's use your variables `user` and `shoppingCart` instead of making new queries to the DB